pages/job-category/index.js

```diff
@@ -0,0 +1,157 @@
+import Link from "next/link";
+import { styled } from "styled-components";
+import JobList from "../components/JobList";
+import { v4 as uuidv4 } from "uuid";
+import tailwindConfig from "@/tailwind.config";
+import postcssConfig from "@/postcss.config";
+import Image from "next/image";
+
+const StyledArticle = styled.article`
+  border-radius: 20px;
+  border: 1px solid lightgrey;
+  margin: 5px;
+  padding: 10px;
+  width: 70%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  font-size: 20px;
+`;
+
+const StyledHeading = styled.h1`
+  text-align: center;
+  color: white;
+`;
+const StyledSpan = styled.span`
+  border-radius: 20px;
+  border: 1px lightgrey;
+  box-shadow: 1px 1px 2px lightgrey;
+  background-color: #c8c8c8;
+  margin: 5px;
+  padding: 10px;
+  width: 400px;
+  display: flex;
+  flex-direction: column;
+
+  align-items: center;
+  font-size: 20px;
+`;
+
+const StyledDiv = styled.div`
```

Comment:
I think we might want to think of a better name for this. 

Maybe something like `StyledWrapper` ?